book/A-git-in-other-environments/sections/powershell.asc

```diff
@@ -28,16 +28,19 @@ If you have at least PowerShell 5 or PowerShell 4 with PackageManagement install
 More information about the requirements: (https://docs.microsoft.com/en-us/powershell/gallery/psget/get_psget_module[])
 [source,powershell]
 -----
+> Set-PSRepository -Name PSGallery -InstallationPolicy Trusted
 > Update-Module PowerShellGet -Force
-> Install-Module Posh-Git -Scope LocalMachine
+> Install-Module Posh-Git -Scope AllUsers
 -----
-If you want to install Posh-Git only for the currnet user and not globaly, use "-Scope CurrentUser" instead.
+If you want to install Posh-Git only for the current user and not globally, use "-Scope CurrentUser" instead. 
+If the 2nd command fails with the message `Update-Module : Module 'PowerShellGet' was not installed by using Install-Module, so it cannot be updated.` You need to run: `Install-Module PowerShellGet -Force -SkipPublisherCheck` instead. This happens, because the modules that ship with Windows Powershell are signed with a different publishment certificate.
```

Comment:
See comment above about one-line-per-sentence formatting. Also, can I tweak the wording a bit?

```
If the second command fails with an error like `Module 'PowerShellGet' was not installed by using Install-Module`, you'll need to run another command first:

[source,powershell]
----
> Install-Module PowerShellGet -Force -SkipPublisherCheck
----

Then you can go back and try again.
```